components/telemetry-operator/internal/webhook/logpipeline_webhook.go

```diff
@@ -73,7 +74,8 @@ func NewLogPipeLineValidator(
 	outputValidator validation.OutputValidator,
 	pipelineConfig fluentbit.PipelineConfig,
 	fsWrapper fs.Wrapper,
-	restartsTotal prometheus.Counter) *LogPipelineValidator {
+	restartsTotal prometheus.Counter,
+	fileValidator validation.FilesValidator) *LogPipelineValidator {
```

Comment:
Let's change the order or the arguments to have all the validators consecutively. Maybe it would also make sense to wrap all validators into some struct to shorten the argument list a bit.